src/index.js

```diff
@@ -8,43 +9,55 @@ const defaultOptions = {
   source: true,
 };
 
-export default {
-  addWithInfo(storyName, info, storyFn, _options) {
-    if (typeof storyFn !== 'function') {
-      if (typeof info === 'function') {
-        _options = storyFn;
-        storyFn = info;
-        info = '';
-      } else {
-        throw new Error('No story defining function has been specified');
-      }
-    }
+const defaultMtrcConf = {
+	h1: H1,
+	h2: H2,
+	h3: H3,
+	h4: H4,
+	h5: H5,
+	h6: H6,
+	code: Code,
+	p: P,
+	a: A,
+	li: LI,
+	ul: UL,
+	};
 
+export default {
+  addWithInfo(storyName, info, storyFn, _options, _mtrcConf) {
     const options = {
       ...defaultOptions,
       ..._options
     };
-
```

Comment:
So [here](https://github.com/kadirahq/react-storybook-addon-info/pull/54/files) you can see the actual changes made in this PR. There are some changes that we don't need for this feature. I think those were made unintentionally when you do a `git pull`. Could clean the PR up a bit by keeping only the changes you intended to make?